Sources/MockoloFramework/Models/ArgumentsHistoryModel.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+final class ArgumentsHistoryModel: Model {
+    var name: String
+    var type: Type
+    var offset: Int64 = .max
+    let suffix: String
+    let capturableParamNames: [String]
+    let capturableParamTypes: [Type]
+    let isHistoryAnnotated: Bool
+
+    var modelType: ModelType {
+        return .class
+    }
+
+    init(name: String, genericTypeParams: [ParamModel], params: [ParamModel], isHistoryAnnotated: Bool, suffix: String) {
+        self.name = name + .argsHistorySuffix
+        self.suffix = suffix
+        self.isHistoryAnnotated = isHistoryAnnotated
+
+        // Value contains closure is not supported.
+        let capturables = params.filter { !$0.type.hasClosure }
+        self.capturableParamNames = capturables.map(path: \.name)
+        self.capturableParamTypes = capturables.map(path: \.type)
+        
+        let genericTypeNameList = genericTypeParams.map(path: \.name)
+        self.type = Type.toArgumentsHistoryType(with: capturableParamTypes, typeParams: genericTypeNameList)
+    }
+    
+    func needsCaptureHistory(force: Bool) -> Bool {
```

Comment:
rename -> func enable(force:)

Reply:
Maybe `enabled(force:)` is better than `enable(force:)`?
Because this return the state.

Reply:
On second thought, `enable(force:)` was more natural.
I will fix it.

Reply:
Fixed in b35d242.